### DISCUSSION TITLE
AKHQ - Not Connecting to KafkaCluster endpoint - [Status: 409]

### DISCUSSION THREAD

**Discussion by Fran-78** [tchiotludo/akhq#1553] [Q&A]
07T09:24:13.189653Z] [Duration: 60002 ms] [Url: GET /api/cluster1/topic] [Status: 409] [Ip: /x.x.x.x] [User: user @ company.com]

Hi, receiving a Status:409 notification in akhq deployment logs, when trying to connect to one of the clusters. 

- eks clusters 1 = namespace 1 - 10 > each namespace has a local kafkacluster.
-  eks cluster 2 = namespace for akhq
- akhq v0.23.0 - can connect to 3 of those kafkaclusters (all of which uses the same config template)
- but for some unknown reason certain connections do no work anymore - they did work fine for a very long time.

each local kafkacluster has a aws ec2 loadbalancer kafkacluster-external.namespace1:19092 TCP > aws ingress ac8899f4560ef4654321e567d740f0f9-345608445.us-west-2.elb.amazonaws.com.

Can anyone tell me how to sort this issue ok, or better best way to trouble shout this issue. How to add Debug logging. I have deployed another busybox pod from same namespace as AKHQ and managed to telnet to kafkaclusters, just to be sure this is working. But im thinking this might AWS related (load balancer) - i have actual deleted the service for a particular kafkacluster that didnt work and redeployed it, so new load balancer for the local kafkacluster endpoint but still no luck.

Anyway guidance would be appreciated- on debugging and approach to resolve this issue.

**Comment by tchiotludo** [comment]
which version are you using at the end? do you tried with latest?

**Comment by zencircle** [comment]
409 and high latency, points to NLB. Try if you put all your clusters and akhq cluster in the same AZ ? 
Run kafkacat on the akhq cluster and connect to all 10 kafka clusters. This might help you isolate if its akhq issue
